src/crimson/os/futurized_store.cc

```diff
@@ -14,9 +14,13 @@ FuturizedStore::create(const std::string& type,
   if (type == "memstore") {
     return std::make_unique<crimson::os::CyanStore>(data);
   } else if (type == "bluestore") {
-    return std::make_unique<crimson::os::AlienStore>(data, values, alien);
+    return std::make_unique<crimson::os::AlienStore>(
+      type, data, values, alien);
   } else if (type == "seastore") {
     return crimson::os::seastore::make_seastore(data, values);
+  } else if (type.rfind("alienstore+", 0) == 0) {
```

Comment:
`std::string::starts_with` is available since C+20, sorry.

Reply:
why do we need to add new store types? i think the plain "bluestore", "memstore" would suffice. instead, how about just adding "cyanstore" to allowed values of "osd_objectstore"?

Reply:
I think the problem is in legacy. We already started expecting `CyanStore` when `memstore` is passed (`--memstore` of `vstart.sh`).

Reply:
i'd prefer fixing this at a higher level -- just add a `--cyanstore` option. and update `doc/dev/crimson/crimson.rst` accordingly. i can do it.

Reply:
Fixed and repushed!

Reply:
> i'd prefer fixing this at a higher level -- just add a `--cyanstore` option. and update `doc/dev/crimson/crimson.rst` accordingly. i can do it.

#42476